napi_js_data.md

```diff
@@ -0,0 +1,760 @@
+# N-API - JavaScript Data Types and Values 
+
+N-API exposes a set of APIs to create fundamental JavaScript Objects. 
```

Comment:
~~fundamental JavaScript Objects~~ all types of JavaScript values.

"Fundamental" sounds too much like it might be referring only to primitive types.